### ISSUE TITLE
Gap between influence and real loss difference

### ISSUE THREAD

**Issue by HAoYifei996** [nimarb/pytorch_influence_functions#18]
Hi, I am trying to perform a leave-one-out retraining to compare the difference between influence computed by `calc_influence_single` [here](https://github.com/nimarb/pytorch_influence_functions/blob/66c9a9e250d663a7fc36d67ccc4cfa589bccc846/pytorch_influence_functions/calc_influence_function.py#L281) and actual loss difference by computing two times of loss before and after retrain. I use `calc_loss`[ here](https://github.com/nimarb/pytorch_influence_functions/blob/66c9a9e250d663a7fc36d67ccc4cfa589bccc846/pytorch_influence_functions/influence_function.py#L57) to compute loss. Also, I scale the influence by `len(trainset)`. Surprisingly I found big difference between computed influence and real loss difference after retrain. For a random pick `test_idx=10, train_idx_to_remove = 609`, I'm getting the following result:

```
actual_loss_diffs:  tensor(0.7848, device='cuda:0', grad_fn=<SubBackward0>)
predict_loss_diffs:  -0.003923072204589844
```

which doesn't seem very relevant to me.

Thanks in advance for any kind suggestions!

**Comment by HAoYifei996** [comment]
So far I think it may have something to do with the net in the example is only trained with 10 epochs, which does not achieve close to global optimum, but I'm not sure.
